### PR TITLE
faq: remove distro-specific examples

### PR DESCRIPTION
**Pull request by miabbott**
We have examples on how to use "bootc compatible" images in other distro specific locations, so just link to those.

### CODE REVIEW

**Review by cgwalters — APPROVED** [review]

**Comment by cgwalters** [comment]
@miabbott needs DCO

File: docs/faq.md [hunk 1]
```diff
@@ -6,34 +6,9 @@ nav_order: 4
...
-RUN yum -y install go-toolset
-RUN go build hello-world.go
-
-FROM quay.io/redhat/rhel-base:10
-COPY --from=builder /build/hello-world /usr/bin
-RUN ostree container commit
-```
-
-You can find more examples at the [centos-boot-layered repo](https://github.com/CentOS/centos-boot-layered) repo or the [CoreOS layering-examples repo](https://github.com/coreos/layering-examples).
+For examples of how use "bootc compatible" container images, see the [centos-boot-layered repo](https://github.com/CentOS/centos-boot-layered) or the [CoreOS layering-examples repo](https://github.com/coreos/layering-examples).
```
[the remark this answers is not in the document]
Comment (cgwalters) [reply docs/faq.md:11]: Ultimately there are going to be multiple distributions using bootc, right?  So a single source-of-truth gets hard.

Now, centos-boot-layered is currently intended for images pre-built by the project that are not "base images".

One immediately obvious problem with trying to unify coreos/layering-examples with centos-boot stuff is that coreos/layering-examples uses rpm-ostree because there is no `dnf` in FCOS today.

[the remark this answers is not in the document]
Comment (vrothberg) [reply docs/faq.md:11]: Fair point.  Let me rephrase.

Assuming I'd be a new user, I prefer to follow one known-to-work workflow.  Being pointed to multiple alternatives may confuse me.  (Based on previous experience,  I am implying that the docs' qualities vary and may not be up to date)

[the remark this answers is not in the document]
Comment (vrothberg) [reply docs/faq.md:11]: What I desire is _one_ known-to-work example.  Pointing to multiple distros and implementations after that point is totally fine.

[the remark this answers is not in the document]
Comment (miabbott) [reply docs/faq.md:11]: How about I modify the text to point to `centos-boot` as an example of building a base image and point to `centos-boot-layered` as an example of derived images?

I feel those are the best options at the moment.

[the remark this answers is not in the document]
Comment (vrothberg) [reply docs/faq.md:11]: Sounds great, thanks @miabbott !

**Review by cgwalters — APPROVED** [review]
